lib/src/validator.dart

```diff
@@ -124,6 +124,9 @@ class Validator {
           if (const {'__404error.html', 'categories.json'}.contains(fullPath)) {
             _warn(PackageWarning.orphanedFile, fullPath, _origin);
           }
+          if (const {'search.html', 'categories.json'}.contains(fullPath)) {
```

Comment:
We can just fold `'search.html'` into the const set on line 124, since this is the same check.

Reply:
Addressed in new commit